internal/daysteps/daysteps.go

```diff
@@ -10,6 +16,22 @@ var (
 
 func parsePackage(data string) (int, time.Duration, error) {
 	// ваш код ниже
+	parts := strings.Split(data, ",")
+	if len(parts) != 2 {
+		return 0, 0, errors.New("Ошибка")
+	}
+	numOfSteps, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return 0, 0, errors.New("Ошибка")
+	}
+
+	duration, err := time.ParseDuration(parts[1])
+	if err != nil {
+		return 0, 0, errors.New("Ошибка")
```

Comment:
Теряешь ошибку, нужно везде исправить, где есть ошибка
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```